src/test/java/io/meeds/qa/ui/stepDefinitions/UserProfileStepDefinition.java

```diff
@@ -377,4 +373,14 @@ public void isProfileContactJobVisible(String job) {
     userProfileSteps.isProfileContactJobVisible(job);
   }
 
+  @Then("^The weekly point chart is displayed$")
```

Comment:
No need to regex when no method parameters

```suggestion
  @Then("The weekly point chart is displayed")
```

Reply:
done